app/api/manager.py

```diff
@@ -19,45 +24,38 @@ def create_user():
         user = data_now_json_str(user_schema)
     except Exception as err:
         return jsonify({"success": False, "message": str(err)}), 400
-    if role_valid(user['role']):
-        message = "Enter a valid role"
-        return jsonify({"success": False, "message": message}), 400
-    if user['role'] == "employee":
-        message = add_user(user)
-        if message is True:
-            return jsonify({"success": True, "message": "Register sucessfully"
-                            }), 200
-        return jsonify({"success": True, "message": message}), 400
-    message = "Manager can add only employee"
-    return jsonify({"success": False, "message": message}), 400
+
+    message, response = add_user(user, "employee")
+    if response:
+        return jsonify({"success": True, "message": "Register sucessfully"}), 200
+    return jsonify({"success": True, "message": message}), 400
 
 
-@manager_bp.route('/view-task', endpoint='view_task', methods=['GET'])
+@manager_bp.route("/view-task", endpoint="view_task", methods=["GET"])
 @manager_required
 def view_task():
-    task_id = request.args.get('task_id', default=None)
+    task_id = request.args.get("task_id", default=None)
     decoded_jwt = token_decode()
-    user_id = decoded_jwt['user_id']
+    user_id = decoded_jwt["user_id"]
     if task_id is None:
-        all_task = list(task_details('assigned_by', user_id))
+        all_task = list(task_details("assigned_by", user_id))
 
         if len(all_task) == 0:
-            return jsonify({"success": False, "message":
-                            "No task is assign"}), 400
+            return jsonify({"success": False, "message": "No task is assign"}), 400
         all_task = serialize_list(all_task)
         return jsonify({"success": True, "details": all_task}), 200
 
     try:
-        task = task_details('_id', ObjectId(task_id))
-        if user_id == task['assigned_by']:
+        task = task_details("_id", ObjectId(task_id))
+        if user_id == task["assigned_by"]:
```

Comment:
better to check this using the query itself instead of the extra if condition.
`if user_id == task["assigned_by"]:`